src/main/perl/TextRender.pm

```diff
@@ -154,16 +153,10 @@ Java properties format (using C<Config::Properties>),
 
 .INI format (using C<Config::Tiny>)
 
-=item general
-
-(using C<Config::General>).
-
-Caution: there is no way to garantee reproducible results as
-there is no control over the order of hash keys. It is recommended to
-try to generate this format in another way (e.g. via TT).
-
 =back
 
+(Builtin module <general> is provided as part of L<CCM::TextRender> since 15.10).
```

Comment:
If general is still available  as a "module" and useable in the same was from a user perspective then it should remain an =item, and yes drop "builtin". Something like:

```
=item general

key = value\n format, provided by L<CCM::TextRender>. See the CCM::TextRender pod for more information.
```